pipeline/src/test_util.rs

```diff
@@ -119,10 +116,18 @@ pub fn run_pilcom_with_backend_variant(
     }
 }
 
-pub fn gen_estark_proof(file_name: &str, inputs: Vec<GoldilocksField>) {
-    let pipeline = make_prepared_pipeline(file_name, inputs, Vec::new());
-    gen_estark_proof_with_backend_variant(pipeline.clone(), BackendVariant::Monolithic);
-    gen_estark_proof_with_backend_variant(pipeline, BackendVariant::Composite);
+fn should_generate_proofs() -> bool {
+    env::var("POWDR_GENERATE_PROOFS")
+        .ok()
+        .map(|v| !v.is_empty())
+        == Some(true)
```

Comment:
So, `POWDR_GENERATE_PROOFS=false` or `0` is true, and `POWDR_GENERATE_PROOFS=""` is false?

I would change to `env::vars_os().any(|(k, _)| k.to_str() == Some("POWDR_GENERATE_PROOFS"))`, so if it is set to whatever, it is true, even empty or invalid UTF-8.

Reply:
But then, can you easily un-define it? Say I have it set to some value in my environment, but I want to turn off proof generation for just one test run?

Maybe we we should only accept `true` and `false` (panic if set to any other value), and default to `false` if it is unset?

Reply:
According to chatgpt, you can do `env -u VARIABLE_NAME command`. But it is settled now.